pixl_core/tests/test_project_config.py

```diff
@@ -30,16 +30,20 @@ def test_config_from_file():
 
     assert project_config.project.name == "test-extract-uclh-omop-cdm"
     assert project_config.project.modalities == ["DX", "CR"]
-    assert project_config.tag_operation_files == [
-        PROJECT_CONFIGS_DIR / "tag-operations" / "test-extract-uclh-omop-cdm.yaml"
-    ]
+    assert (
+        project_config.tag_operation_files.base
+        == PROJECT_CONFIGS_DIR / "tag-operations" / "test-extract-uclh-omop-cdm.yaml"
+    )
     assert project_config.destination.dicom == "ftps"
     assert project_config.destination.parquet == "ftps"
 
 
 BASE_YAML_DATA = {
     "project": {"name": "myproject", "modalities": ["DX", "CR"]},
-    "tag_operations": ["test-extract-uclh-omop-cdm.yaml"],
+    "tag_operations": {
+        "base": "test-extract-uclh-omop-cdm.yaml",
+        "manufacturer_overrides": "mri-diffusion.yaml",
+    },
     "destination": {"dicom": "ftps", "parquet": "ftps"},
 }
 
```

Comment:
Feels strange that we're not testing a load config method here for the overrides and that its in main instead